reviewers.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict
+import operator
+import os
+import re
+
+
+def prompt_for_user():
+    while True:
+        try:
+            user_input = input("\nName or email (case insensitive): ") 
+        except (KeyboardInterrupt, EOFError):
+            return None
+        clean_input = user_input.strip().lower()
+        if clean_input != "":
+            return clean_input
+
+
+if __name__ == "__main__":
+    print("Utility to help generate 'Reviewers' string for Pull Requests. Use Ctrl+D or Ctrl+C to exit")
+    
+    stream = os.popen("git log | grep Reviewers")
```

Comment:
How about this since you are `lstrip("Reviewers: ")` later in the code?
```python
    stream = os.popen('git log --grep "Reviewers: " --format=%b | grep "Reviewers: "')
```